tests/evm/test_error_stack.py

```diff
@@ -0,0 +1,159 @@
+import pytest
+from collections import namedtuple
+from itertools import chain
+
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    CallContextFieldTag,
+    Block,
+    Transaction,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC
+
+BYTECODE = Bytecode().pop()
+BYTECODE_PUSH = Bytecode().push1(0x10).push1(0x20)
+
```

Comment:
Could you add another test vector that uses `ADD`? (or any other opcode that has more pops than pushes to the stack)

Reply:
add `ADD` in [e47e611](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/301/commits/e47e611ba0917d0e60b5ddc43db2742f89541b3b) thanks !